### PR TITLE
Fix e2e test lifecycle/ha_master.go

### PR DESCRIPTION
**Pull request by anjensan**
Minor fixes for gce-ha-master e2e test
https://k8s-testgrid.appspot.com/google-gce#gce-ha-master

```release-note
NONE
```

### CODE REVIEW

File: test/e2e/lifecycle/ha_master.go [hunk 1]
```diff
@@ -78,7 +78,7 @@ func createNewRC(c clientset.Interface, ns string, name string) {
 }
 
 func findRegionForZone(zone string) string {
-	region, err := exec.Command("gcloud", "compute", "zones", "list", zone, "--quiet", "--format=csv[no-heading](region)").CombinedOutput()
+	region, err := exec.Command("gcloud", "compute", "zones", "list", zone, "--quiet", "--format=csv[no-heading](region)").Output()
```
Comment (wojtek-t) [inline test/e2e/lifecycle/ha_master.go]: why?

Comment (anjensan) [reply test/e2e/lifecycle/ha_master.go]: gcloud may print additional information into 'stderr' (warnings & logs).

For example after setting 'gcloud config set core/verbosity ...' (which I used to debug skew-master tests for example) there is no chance for this test to pass (CombinedOutput always returns "garbage").

NOTE: .Output() also captures 'stderr' (last 32k) when return code != 0.

**Pull request merged**